compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1564,6 +1564,15 @@ static LogicalResult setRootConfig(
   SmallVector<int64_t> ubs = linalgOp.getStaticLoopRanges();
   auto translationInfo = IREE::Codegen::TranslationInfoAttr::get(
       entryPointFn->getContext(), pipeline);
+
+  // Always vectorize the ops for VMVX pipeline because stack allocation is not
+  // allowed.
+  if (pipeline == DispatchLoweringPassPipeline::VMVXDefault) {
+    for (int i = 0, e = ubs.size(); i < e; ++i) {
+      if (ubs[i] == ShapedType::kDynamic) ubs[i] = 1;
+    }
+  }
```

Comment:
I think we have a utility somewhere doing something like that but I'm not sure if it would work for this case. If so, it would be great to use it to have all the "always vectorize" cases going through the same code.

Reply:
Yes, I know what you're referring. We have `setAlwaysVectorizeSizes`, but that's for multi-level tiling and is specific for the last level of tiling. For default linalg ops, it seems that we only apply one level of tiling (which is tile+distribute). So it does not work for this case.

```
static void setAlwaysVectorizeSizes(linalg::LinalgOp op,
                                    SmallVectorImpl<int64_t> &parallelSizes,
                                    SmallVectorImpl<int64_t> &reductionSizes) {
```